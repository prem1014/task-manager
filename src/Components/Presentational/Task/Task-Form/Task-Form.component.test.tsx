import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import TaskForm from './Task-Form.component';

describe('TaskForm component test suite', () => {
    Enzyme.configure({ adapter: new Adapter() });
    let componentWrapper: any;
    const taskFormProps = {
        taskListNameCreated: jest.fn()
    }

    beforeEach( () => {
        componentWrapper = shallow(<TaskForm {...taskFormProps}/>);
    });

    it('TaskForm component snapshot test', () => {
        expect(componentWrapper).toMatchSnapshot();
    });
});