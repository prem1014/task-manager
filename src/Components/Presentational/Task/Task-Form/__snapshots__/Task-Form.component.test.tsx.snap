// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskForm component test suite TaskForm component snapshot test 1`] = `ShallowWrapper {}`;
